### ISSUE TITLE
Crash with Doom 2 on reaching goal.

### ISSUE THREAD

**Issue by RealRexen** [Daivuk/apdoom#13]
APDoom 1.0.3 had a crash with doom 2 on reaching the goal. 
These are my settings:
![image](https://github.com/Daivuk/apdoom/assets/18408903/32338be5-a5f9-4566-8403-3b6d7d6c831b)

**Comment by RealRexen** [comment]
Here's the archipelago log just in case it helps 

```
[root at 2024-05-02 16:59:32,566]: Archipelago (0.4.6) logging initialized on Linux-5.15.0-105-generic-x86_64-with-glibc2.35 running Python 3.10.12
[websockets.server at 2024-05-02 16:59:32,705]: server listening on 0.0.0.0:54256
[websockets.server at 2024-05-02 16:59:32,709]: server listening on [::]:54256
[root at 2024-05-02 16:59:32,709]: Hosting game at archipelago.gg:54256
[websockets.server at 2024-05-02 17:00:18,103]: connection open
[root at 2024-05-02 17:00:18,264]: Notice (all): Rexen (Team #1) playing DOOM II has joined. Client(0.4.1), [].
[root at 2024-05-02 17:00:18,264]: Notice (Player Rexen in team 1): Now that you are connected, you can use !help to list commands to run via the server. If your client supports it, you may have additional local commands you can list with /help.
[root at 2024-05-02 17:00:42,621]: (Team #1) Rexen sent Super Shotgun to Rexen (Entryway (MAP01) - Chainsaw)
[root at 2024-05-02 17:01:18,446]: (Team #1) Rexen sent The Gantlet (MAP03) to Rexen (Entryway (MAP01) - Shotgun)
[root at 2024-05-02 17:01:25,176]: (Team #1) Rexen sent Plasma gun to Rexen (Entryway (MAP01) - Rocket launcher)
[root at 2024-05-02 17:01:39,821]: (Team #1) Rexen sent Armor to Rexen (Entryway (MAP01) - Armor)
[root at 2024-05-02 17:01:58,822]: (Team #1) Rexen sent Entryway (MAP01) - Complete to Rexen (Entryway (MAP01) - Exit)
[root at 2024-05-02 17:02:05,539]: (Team #1) Rexen sent Berserk to Rexen (The Gantlet (MAP03) - Shotgun)
[root at 2024-05-02 17:02:48,075]: (Team #1) Rexen sent Tricks and Traps (MAP08) to Rexen (The Gantlet (MAP03) - Mega Armor)
[root at 2024-05-02 17:03:21,221]: (Team #1) Rexen sent Berserk to Rexen (Tricks and Traps (MAP08) - Rocket launcher)
[root at 2024-05-02 17:03:48,074]: (Team #1) Rexen sent Energy cell pack to Rexen (Tricks and Traps (MAP08) - Armor)
[root at 2024-05-02 17:04:05,452]: (Team #1) Rexen sent Box of rockets to Rexen (Tricks and Traps (MAP08) - Plasma gun)
[root at 2024-05-02 17:04:14,820]: (Team #1) Rexen sent Box of bullets to Rexen (Tricks and Traps (MAP08) - Chainsaw)
[root at 2024-05-02 17:04:20,481]: (Team #1) Rexen sent Partial invisibility to Rexen (Tricks and Traps (MAP08) - Armor 2)
[root at 2024-05-02 17:04:48,082]: (Team #1) Rexen sent Box of shotgun shells to Rexen (Tricks and Traps (MAP08) - Invulnerability)
[root at 2024-05-02 17:04:48,312]: (Team #1) Rexen sent Armor to Rexen (Tricks and Traps (MAP08) - Invulnerability 2)
[root at 2024-05-02 17:04:49,437]: (Team #1) Rexen sent Energy cell pack to Rexen (Tricks and Traps (MAP08) - Invulnerability 3)
[root at 2024-05-02 17:04:50,743]: (Team #1) Rexen sent Medikit to Rexen (Tricks and Traps (MAP08) - Invulnerability 4)
[root at 2024-05-02 17:04:57,397]: (Team #1) Rexen sent Box of shotgun shells to Rexen (Tricks and Traps (MAP08) - Supercharge)
[root at 2024-05-02 17:05:19,907]: (Team #1) Rexen sent Energy cell pack to Rexen (Tricks and Traps (MAP08) - Yellow skull key)
[root at 2024-05-02 17:05:29,890]: (Team #1) Rexen sent Box of shotgun shells to Rexen (Tricks and Traps (MAP08) - Shotgun)
[root at 2024-05-02 17:05:44,722]: Notice (all): Rexen (Team #1) has left the game
[websockets.server at 2024-05-02 17:05:44,722]: connection closed
[websockets.server at 2024-05-02 17:05:45,836]: connection open
[websockets.server at 2024-05-02 17:05:46,081]: connection closed
[websockets.server at 2024-05-02 17:05:47,702]: connection open
[root at 2024-05-02 17:05:48,080]: Notice (all): Rexen (Team #1) playing DOOM II has joined. Client(0.4.1), [].
[root at 2024-05-02 17:05:48,081]: Notice (Player Rexen in team 1): Now that you are connected, you can use !help to list commands to run via the server. If your client supports it, you may have additional local commands you can list with /help.
[root at 2024-05-02 17:06:40,866]: (Team #1) Rexen sent Box of shotgun shells to Rexen (Tricks and Traps (MAP08) - Supercharge 2)
[root at 2024-05-02 17:07:09,224]: (Team #1) Rexen sent Armor to Rexen (Tricks and Traps (MAP08) - BFG9000)
[root at 2024-05-02 17:07:12,764]: (Team #1) Rexen sent The Crusher (MAP06) to Rexen (Tricks and Traps (MAP08) - Backpack)
[root at 2024-05-02 17:07:14,667]: (Team #1) Rexen sent BFG9000 to Rexen (Tricks and Traps (MAP08) - Backpack 2)
[root at 2024-05-02 17:07:16,309]: (Team #1) Rexen sent Supercharge to Rexen (Tricks and Traps (MAP08) - Backpack 5)
[root at 2024-05-02 17:07:17,023]: (Team #1) Rexen sent Armor to Rexen (Tricks and Traps (MAP08) - Backpack 3)
[root at 2024-05-02 17:07:19,395]: (Team #1) Rexen sent Armor to Rexen (Tricks and Traps (MAP08) - Backpack 4)
[root at 2024-05-02 17:07:45,447]: (Team #1) Rexen sent Supercharge to Rexen (Tricks and Traps (MAP08) - Chaingun)
[root at 2024-05-02 17:08:29,995]: (Team #1) Rexen sent Partial invisibility to Rexen (Tricks and Traps (MAP08) - Partial invisibility)
[root at 2024-05-02 17:09:29,423]: (Team #1) Rexen sent Box of rockets to Rexen (The Gantlet (MAP03) - Mega Armor 2)
[root at 2024-05-02 17:09:31,053]: (Team #1) Rexen sent Shotgun to Rexen (The Gantlet (MAP03) - Supercharge)
[root at 2024-05-02 17:09:33,943]: (Team #1) Rexen sent Chaingun to Rexen (The Gantlet (MAP03) - Partial invisibility)
[root at 2024-05-02 17:09:36,456]: (Team #1) Rexen sent Mega Armor to Rexen (The Gantlet (MAP03) - Rocket launcher)
[root at 2024-05-02 17:09:45,206]: (Team #1) Rexen sent Medikit to Rexen (The Gantlet (MAP03) - Chaingun)
[root at 2024-05-02 17:09:47,449]: (Team #1) Rexen sent Supercharge to Rexen (The Gantlet (MAP03) - Blue keycard)
[root at 2024-05-02 17:10:55,368]: (Team #1) Rexen sent Partial invisibility to Rexen (The Crusher (MAP06) - Megasphere)
[root at 2024-05-02 17:11:05,334]: (Team #1) Rexen sent The Waste Tunnels (MAP05) to Rexen (The Crusher (MAP06) - Blue keycard 2)
[root at 2024-05-02 17:11:06,306]: (Team #1) Rexen sent The Waste Tunnels (MAP05) - Yellow keycard to Rexen (The Crusher (MAP06) - Plasma gun)
[root at 2024-05-02 17:11:19,251]: (Team #1) Rexen sent Medikit to Rexen (The Crusher (MAP06) - Blue keycard 3)
[root at 2024-05-02 17:11:22,828]: (Team #1) Rexen sent Energy cell pack to Rexen (The Crusher (MAP06) - Blue keycard)
[root at 2024-05-02 17:11:32,940]: (Team #1) Rexen sent Circle of Death (MAP11) to Rexen (The Crusher (MAP06) - Supercharge)
[root at 2024-05-02 17:11:48,371]: (Team #1) Rexen sent The Crusher (MAP06) - Red keycard to Rexen (The Crusher (MAP06) - Armor)
[root at 2024-05-02 17:12:05,230]: (Team #1) Rexen sent Box of bullets to Rexen (The Waste Tunnels (MAP05) - Super Shotgun)
[root at 2024-05-02 17:12:06,763]: (Team #1) Rexen sent Refueling Base (MAP10) to Rexen (The Waste Tunnels (MAP05) - Shotgun)
[root at 2024-05-02 17:12:09,856]: (Team #1) Rexen sent Energy cell pack to Rexen (The Waste Tunnels (MAP05) - Supercharge 2)
[root at 2024-05-02 17:12:14,908]: (Team #1) Rexen sent Circle of Death (MAP11) - Blue keycard to Rexen (The Waste Tunnels (MAP05) - Armor)
[root at 2024-05-02 17:13:13,794]: (Team #1) Rexen sent Box of bullets to Rexen (The Waste Tunnels (MAP05) - Mega Armor)
[root at 2024-05-02 17:13:33,139]: (Team #1) Rexen sent Medikit to Rexen (The Waste Tunnels (MAP05) - Supercharge)
[root at 2024-05-02 17:13:34,396]: (Team #1) Rexen sent Box of bullets to Rexen (The Waste Tunnels (MAP05) - Berserk)
[root at 2024-05-02 17:13:58,108]: (Team #1) Rexen sent Box of shotgun shells to Rexen (The Waste Tunnels (MAP05) - Red keycard)
[root at 2024-05-02 17:14:16,139]: (Team #1) Rexen sent Refueling Base (MAP10) - Yellow keycard to Rexen (The Waste Tunnels (MAP05) - Rocket launcher)
[root at 2024-05-02 17:15:12,421]: (Team #1) Rexen sent The Focus (MAP04) - Blue keycard to Rexen (Refueling Base (MAP10) - Shotgun)
[root at 2024-05-02 17:15:22,942]: (Team #1) Rexen sent Dead Simple (MAP07) to Rexen (Refueling Base (MAP10) - Armor)
[root at 2024-05-02 17:15:44,727]: (Team #1) Rexen sent Medikit to Rexen (Refueling Base (MAP10) - Chaingun)
[root at 2024-05-02 17:16:52,686]: (Team #1) Rexen sent Box of shotgun shells to Rexen (Refueling Base (MAP10) - Mega Armor)
[root at 2024-05-02 17:17:20,281]: (Team #1) Rexen sent Backpack to Rexen (Refueling Base (MAP10) - Yellow keycard)
[root at 2024-05-02 17:17:38,291]: (Team #1) Rexen sent Box of rockets to Rexen (Refueling Base (MAP10) - Backpack)
[root at 2024-05-02 17:17:51,797]: (Team #1) Rexen sent Armor to Rexen (Refueling Base (MAP10) - Supercharge 2)
[root at 2024-05-02 17:18:11,631]: (Team #1) Rexen sent Partial invisibility to Rexen (Refueling Base (MAP10) - Berserk)
[root at 2024-05-02 17:18:24,540]: (Team #1) Rexen sent Invulnerability to Rexen (Refueling Base (MAP10) - Invulnerability)
[root at 2024-05-02 17:18:55,720]: (Team #1) Rexen sent Box of rockets to Rexen (Refueling Base (MAP10) - Chainsaw)
[root at 2024-05-02 17:18:59,596]: (Team #1) Rexen sent Energy cell pack to Rexen (Refueling Base (MAP10) - Megasphere)
[root at 2024-05-02 17:20:19,171]: (Team #1) Rexen sent Underhalls (MAP02) - Red keycard to Rexen (Refueling Base (MAP10) - BFG9000)
[root at 2024-05-02 17:20:25,311]: (Team #1) Rexen sent Box of rockets to Rexen (Refueling Base (MAP10) - Blue keycard)
[root at 2024-05-02 17:20:45,701]: (Team #1) Rexen sent The Pit (MAP09) - Blue keycard to Rexen (Refueling Base (MAP10) - Partial invisibility)
[root at 2024-05-02 17:20:52,286]: (Team #1) Rexen sent Energy cell pack to Rexen (Refueling Base (MAP10) - Rocket launcher)
[root at 2024-05-02 17:20:52,525]: (Team #1) Rexen sent Refueling Base (MAP10) - Blue keycard to Rexen (Refueling Base (MAP10) - Armor 2)
[root at 2024-05-02 17:21:10,456]: (Team #1) Rexen sent Mega Armor to Rexen (Refueling Base (MAP10) - Berserk 2)
[root at 2024-05-02 17:21:24,029]: (Team #1) Rexen sent Wolfenstein2 (MAP31) to Rexen (Refueling Base (MAP10) - Plasma gun)
[root at 2024-05-02 17:21:24,453]: (Team #1) Rexen sent Box of bullets to Rexen (Refueling Base (MAP10) - Supercharge)
[root at 2024-05-02 17:21:43,800]: (Team #1) Rexen sent Medikit to Rexen (Refueling Base (MAP10) - Invulnerability 2)
[root at 2024-05-02 17:21:57,815]: (Team #1) Rexen sent Refueling Base (MAP10) - Complete to Rexen (Refueling Base (MAP10) - Exit)
[root at 2024-05-02 17:22:07,567]: (Team #1) Rexen sent Supercharge to Rexen (Wolfenstein2 (MAP31) - Chaingun)
[root at 2024-05-02 17:22:14,470]: (Team #1) Rexen sent Supercharge to Rexen (Wolfenstein2 (MAP31) - Partial invisibility)
[root at 2024-05-02 17:22:40,561]: (Team #1) Rexen sent The Pit (MAP09) - Yellow keycard to Rexen (Wolfenstein2 (MAP31) - BFG9000)
[root at 2024-05-02 17:22:42,822]: (Team #1) Rexen sent Berserk to Rexen (Wolfenstein2 (MAP31) - Megasphere)
[root at 2024-05-02 17:22:45,063]: (Team #1) Rexen sent Box of bullets to Rexen (Wolfenstein2 (MAP31) - Backpack)
[root at 2024-05-02 17:22:45,088]: (Team #1) Rexen sent Supercharge to Rexen (Wolfenstein2 (MAP31) - Backpack 2)
[root at 2024-05-02 17:22:45,206]: (Team #1) Rexen sent Energy cell pack to Rexen (Wolfenstein2 (MAP31) - Backpack 3)
[root at 2024-05-02 17:22:45,683]: (Team #1) Rexen sent Energy cell pack to Rexen (Wolfenstein2 (MAP31) - Backpack 4)
[root at 2024-05-02 17:22:47,571]: (Team #1) Rexen sent Box of shotgun shells to Rexen (Wolfenstein2 (MAP31) - Super Shotgun)
[root at 2024-05-02 17:23:26,456]: (Team #1) Rexen sent The Focus (MAP04) - Yellow keycard to Rexen (Wolfenstein2 (MAP31) - Rocket launcher)
[root at 2024-05-02 17:23:37,944]: (Team #1) Rexen sent Energy cell pack to Rexen (Wolfenstein2 (MAP31) - Shotgun)
[root at 2024-05-02 17:23:59,769]: (Team #1) Rexen sent Armor to Rexen (Wolfenstein2 (MAP31) - Plasma gun)
[root at 2024-05-02 17:24:17,753]: (Team #1) Rexen sent Supercharge to Rexen (Wolfenstein2 (MAP31) - Berserk)
[root at 2024-05-02 17:24:40,029]: (Team #1) Rexen sent Partial invisibility to Rexen (Wolfenstein2 (MAP31) - Supercharge)
[root at 2024-05-02 17:25:07,181]: (Team #1) Rexen sent Box of shotgun shells to Rexen (Dead Simple (MAP07) - Backpack)
[root at 2024-05-02 17:25:08,685]: (Team #1) Rexen sent Underhalls (MAP02) to Rexen (Dead Simple (MAP07) - Super Shotgun)
[root at 2024-05-02 17:25:13,321]: (Team #1) Rexen sent Medikit to Rexen (Dead Simple (MAP07) - Partial invisibility)
[root at 2024-05-02 17:25:16,101]: (Team #1) Rexen sent The Waste Tunnels (MAP05) - Red keycard to Rexen (Dead Simple (MAP07) - Partial invisibility 3)
[root at 2024-05-02 17:25:18,183]: (Team #1) Rexen sent The Crusher (MAP06) - Yellow keycard to Rexen (Dead Simple (MAP07) - Partial invisibility 4)
[root at 2024-05-02 17:25:21,385]: (Team #1) Rexen sent Armor to Rexen (Dead Simple (MAP07) - Partial invisibility 2)
[root at 2024-05-02 17:25:25,957]: (Team #1) Rexen sent Medikit to Rexen (Dead Simple (MAP07) - Megasphere)
[root at 2024-05-02 17:25:29,775]: (Team #1) Rexen sent Mega Armor to Rexen (Dead Simple (MAP07) - Plasma gun)
[root at 2024-05-02 17:25:34,559]: (Team #1) Rexen sent Chainsaw to Rexen (Dead Simple (MAP07) - Berserk)
[root at 2024-05-02 17:25:35,175]: (Team #1) Rexen sent Armor to Rexen (Dead Simple (MAP07) - Rocket launcher)
[root at 2024-05-02 17:25:38,149]: (Team #1) Rexen sent Energy cell pack to Rexen (Dead Simple (MAP07) - Chaingun)
[root at 2024-05-02 17:26:19,514]: Notice (all): Rexen: IDCLIP
[root at 2024-05-02 17:26:41,070]: (Team #1) Rexen sent Dead Simple (MAP07) - Complete to Rexen (Dead Simple (MAP07) - Exit)
[root at 2024-05-02 17:27:24,156]: (Team #1) Rexen sent Tricks and Traps (MAP08) - Yellow skull key to Rexen (Underhalls (MAP02) - Red keycard)
[root at 2024-05-02 17:27:33,132]: (Team #1) Rexen sent Medikit to Rexen (Underhalls (MAP02) - Mega Armor)
[root at 2024-05-02 17:27:50,736]: (Team #1) Rexen sent Medikit to Rexen (Underhalls (MAP02) - Super Shotgun)
[root at 2024-05-02 17:28:25,454]: (Team #1) Rexen sent Box of rockets to Rexen (Underhalls (MAP02) - Blue keycard)
[root at 2024-05-02 17:29:19,847]: (Team #1) Rexen sent Box of shotgun shells to Rexen (The Waste Tunnels (MAP05) - Blue keycard)
[root at 2024-05-02 17:29:31,318]: (Team #1) Rexen sent Energy cell pack to Rexen (The Waste Tunnels (MAP05) - Plasma gun)
[root at 2024-05-02 17:30:03,976]: (Team #1) Rexen sent Box of rockets to Rexen (Tricks and Traps (MAP08) - Red skull key)
[root at 2024-05-02 17:30:13,517]: (Team #1) Rexen sent Mega Armor to Rexen (Tricks and Traps (MAP08) - Invulnerability 5)
[root at 2024-05-02 17:30:30,089]: (Team #1) Rexen sent Armor to Rexen (Circle of Death (MAP11) - Armor)
[root at 2024-05-02 17:30:45,174]: (Team #1) Rexen sent Box of rockets to Rexen (Circle of Death (MAP11) - Chaingun)
[root at 2024-05-02 17:30:55,382]: (Team #1) Rexen sent The Gantlet (MAP03) - Blue keycard to Rexen (Circle of Death (MAP11) - Partial invisibility)
[root at 2024-05-02 17:30:57,747]: (Team #1) Rexen sent Mega Armor to Rexen (Circle of Death (MAP11) - Shotgun)
[root at 2024-05-02 17:31:28,289]: (Team #1) Rexen sent Box of bullets to Rexen (The Gantlet (MAP03) - Backpack)
[root at 2024-05-02 17:31:43,291]: (Team #1) Rexen sent Invulnerability to Rexen (The Gantlet (MAP03) - Red keycard)
[root at 2024-05-02 17:32:33,527]: (Team #1) Rexen sent The Pit (MAP09) to Rexen (Circle of Death (MAP11) - Mega Armor)
[root at 2024-05-02 17:32:53,124]: (Team #1) Rexen sent Underhalls (MAP02) - Blue keycard to Rexen (Circle of Death (MAP11) - Red keycard)
[root at 2024-05-02 17:33:06,272]: (Team #1) Rexen sent Box of shotgun shells to Rexen (Circle of Death (MAP11) - Blue keycard)
[root at 2024-05-02 17:33:07,298]: (Team #1) Rexen sent Energy cell pack to Rexen (Circle of Death (MAP11) - Plasma gun)
[root at 2024-05-02 17:34:32,095]: (Team #1) Rexen sent The Focus (MAP04) to Rexen (Circle of Death (MAP11) - Supercharge)
[root at 2024-05-02 17:35:44,358]: (Team #1) Rexen sent Medikit to Rexen (The Focus (MAP04) - Blue keycard)
[root at 2024-05-02 17:35:51,125]: (Team #1) Rexen sent Box of shotgun shells to Rexen (The Focus (MAP04) - Super Shotgun)
[root at 2024-05-02 17:36:06,491]: (Team #1) Rexen sent Medikit to Rexen (The Focus (MAP04) - Red keycard)
[root at 2024-05-02 17:36:27,520]: (Team #1) Rexen sent Underhalls (MAP02) - Complete to Rexen (Underhalls (MAP02) - Exit)
[root at 2024-05-02 17:36:43,325]: (Team #1) Rexen sent Energy cell pack to Rexen (The Pit (MAP09) - Mega Armor)
[root at 2024-05-02 17:36:47,516]: (Team #1) Rexen sent Energy cell pack to Rexen (The Pit (MAP09) - Shotgun)
[root at 2024-05-02 17:36:58,266]: (Team #1) Rexen sent Mega Armor to Rexen (The Pit (MAP09) - Berserk 2)
[root at 2024-05-02 17:37:32,580]: (Team #1) Rexen sent Box of rockets to Rexen (The Pit (MAP09) - BFG9000)
[root at 2024-05-02 17:37:56,063]: (Team #1) Rexen sent The Gantlet (MAP03) - Red keycard to Rexen (The Pit (MAP09) - Berserk)
[root at 2024-05-02 17:38:13,755]: (Team #1) Rexen sent Medikit to Rexen (The Pit (MAP09) - Blue keycard)
[root at 2024-05-02 17:38:18,013]: (Team #1) Rexen sent The Focus (MAP04) - Red keycard to Rexen (The Pit (MAP09) - Armor)
[root at 2024-05-02 17:38:28,866]: (Team #1) Rexen sent Armor to Rexen (The Pit (MAP09) - Supercharge)
[root at 2024-05-02 17:38:36,400]: (Team #1) Rexen sent Grosse2 (MAP32) to Rexen (The Pit (MAP09) - Rocket launcher)
[root at 2024-05-02 17:39:07,218]: (Team #1) Rexen sent Armor to Rexen (The Pit (MAP09) - Backpack)
[root at 2024-05-02 17:39:09,008]: (Team #1) Rexen sent Box of shotgun shells to Rexen (The Pit (MAP09) - Computer area map)
[root at 2024-05-02 17:39:31,291]: (Team #1) Rexen sent Box of rockets to Rexen (The Pit (MAP09) - Berserk 3)
[root at 2024-05-02 17:39:31,588]: (Team #1) Rexen sent Partial invisibility to Rexen (The Pit (MAP09) - Mega Armor 2)
[root at 2024-05-02 17:39:31,892]: (Team #1) Rexen sent The Waste Tunnels (MAP05) - Blue keycard to Rexen (The Pit (MAP09) - Supercharge 2)
[root at 2024-05-02 17:39:53,209]: (Team #1) Rexen sent Box of shotgun shells to Rexen (The Pit (MAP09) - Yellow keycard)
[root at 2024-05-02 17:40:37,297]: (Team #1) Rexen sent The Pit (MAP09) - Complete to Rexen (The Pit (MAP09) - Exit)
[root at 2024-05-02 17:40:54,021]: (Team #1) Rexen sent Box of rockets to Rexen (Grosse2 (MAP32) - Berserk)
[root at 2024-05-02 17:40:54,462]: (Team #1) Rexen sent Berserk to Rexen (Grosse2 (MAP32) - Rocket launcher)
[root at 2024-05-02 17:41:05,358]: (Team #1) Rexen sent Box of rockets to Rexen (Grosse2 (MAP32) - Chaingun 2)
[root at 2024-05-02 17:41:05,394]: (Team #1) Rexen sent Invulnerability to Rexen (Grosse2 (MAP32) - Chaingun)
[root at 2024-05-02 17:41:05,599]: (Team #1) Rexen sent Box of rockets to Rexen (Grosse2 (MAP32) - Chaingun 3)
[root at 2024-05-02 17:41:24,636]: (Team #1) Rexen sent Box of rockets to Rexen (Grosse2 (MAP32) - Super Shotgun)
[root at 2024-05-02 17:41:29,379]: (Team #1) Rexen sent Tricks and Traps (MAP08) - Red skull key to Rexen (Grosse2 (MAP32) - Megasphere)
[root at 2024-05-02 17:41:38,953]: (Team #1) Rexen sent The Crusher (MAP06) - Blue keycard to Rexen (Grosse2 (MAP32) - Plasma gun)
[root at 2024-05-02 17:41:46,820]: (Team #1) Rexen sent Mega Armor to Rexen (Grosse2 (MAP32) - BFG9000)
[root at 2024-05-02 17:41:52,016]: (Team #1) Rexen sent Box of rockets to Rexen (Grosse2 (MAP32) - Invulnerability)
[root at 2024-05-02 17:42:12,694]: (Team #1) Rexen sent Grosse2 (MAP32) - Complete to Rexen (Grosse2 (MAP32) - Exit)
[root at 2024-05-02 17:42:56,279]: (Team #1) Rexen sent Wolfenstein2 (MAP31) - Complete to Rexen (Wolfenstein2 (MAP31) - Exit)
[root at 2024-05-02 17:43:28,827]: (Team #1) Rexen sent The Gantlet (MAP03) - Complete to Rexen (The Gantlet (MAP03) - Exit)
[root at 2024-05-02 17:43:38,271]: (Team #1) Rexen sent Box of bullets to Rexen (The Focus (MAP04) - Yellow keycard)
[root at 2024-05-02 17:43:49,525]: (Team #1) Rexen sent The Focus (MAP04) - Complete to Rexen (The Focus (MAP04) - Exit)
[root at 2024-05-02 17:44:19,836]: (Team #1) Rexen sent Rocket launcher to Rexen (The Waste Tunnels (MAP05) - Yellow keycard)
[root at 2024-05-02 17:44:39,495]: (Team #1) Rexen sent The Waste Tunnels (MAP05) - Complete to Rexen (The Waste Tunnels (MAP05) - Exit)
[root at 2024-05-02 17:45:46,874]: (Team #1) Rexen sent Medikit to Rexen (The Crusher (MAP06) - Super Shotgun)
[root at 2024-05-02 17:46:20,435]: (Team #1) Rexen sent Energy cell pack to Rexen (The Crusher (MAP06) - Megasphere 2)
[root at 2024-05-02 17:46:21,726]: (Team #1) Rexen sent Mega Armor to Rexen (The Crusher (MAP06) - Backpack)
[root at 2024-05-02 17:46:22,765]: (Team #1) Rexen sent Energy cell pack to Rexen (The Crusher (MAP06) - Rocket launcher)
[root at 2024-05-02 17:46:23,800]: (Team #1) Rexen sent Box of shotgun shells to Rexen (The Crusher (MAP06) - Mega Armor)
[root at 2024-05-02 17:46:36,246]: (Team #1) Rexen sent Circle of Death (MAP11) - Red keycard to Rexen (The Crusher (MAP06) - Red keycard)
[root at 2024-05-02 17:47:06,624]: (Team #1) Rexen sent Supercharge to Rexen (The Crusher (MAP06) - Yellow keycard)
[root at 2024-05-02 17:47:18,391]: (Team #1) Rexen sent The Crusher (MAP06) - Complete to Rexen (The Crusher (MAP06) - Exit)
[root at 2024-05-02 17:47:31,416]: (Team #1) Rexen sent Tricks and Traps (MAP08) - Complete to Rexen (Tricks and Traps (MAP08) - Exit)
[root at 2024-05-02 17:47:52,848]: (Team #1) Rexen sent Medikit to Rexen (Circle of Death (MAP11) - Invulnerability)
[root at 2024-05-02 17:48:06,069]: (Team #1) Rexen sent Box of shotgun shells to Rexen (Circle of Death (MAP11) - Rocket launcher)
[root at 2024-05-02 17:48:09,837]: (Team #1) Rexen sent Supercharge to Rexen (Circle of Death (MAP11) - Backpack)
[root at 2024-05-02 17:48:41,240]: (Team #1) Rexen sent Circle of Death (MAP11) - Complete to Rexen (Circle of Death (MAP11) - Exit)
[root at 2024-05-02 17:48:41,286]: Notice (all): Rexen (Team #1) has completed their goal.
[root at 2024-05-02 17:48:41,287]: Notice (all): Team #1 has completed all of their games! Congratulations!
[root at 2024-05-02 17:48:47,203]: Notice (all): Rexen (Team #1) has left the game
[websockets.server at 2024-05-02 17:48:47,203]: connection closed
```

**Comment by Migs3** [comment]
Pretty sure this was fixed in 1.0.4

**Comment by RealRexen** [comment]
Tbh I wasnt aware of the pre-release versions, I still thought 1.0.3 was the latest version. But I still gave 1.1.0 a shot and it seems to have been fix
